src/Components/CollectionCard/CollectionCardModal.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Bin from '../../assets/icons/Bin.png';
+import Tag from '../../assets/icons/price-tag 1.png';
+import Star from '../../assets/icons/Star1.png';
+import Close from '../../assets/icons/Close.png';
+import Img from '../../assets/icons/image 22.png';
+
+export default function CollectionCardModal({ card, type }) {
```

Comment:
pass the showModal state here too you will need it to show the modal

Reply:
when I created the modals for the spread page 
I did not need to pass the show modal since i will show the modal in the page 
but I did pass setShowModal because we needed it inside the modal to close it